### ISSUE TITLE
bug(YouTube/seekbar) :When "Accessibility Control" is turned on, the bottom half of the seekbar is cut off, and some of its functions cannot be used.

### ISSUE THREAD

**Issue by harutoku** [inotia00/ReVanced_Extended#2415]
### Type

Error at runtime

### Tools used

Revancify

### Application

YouTube v19.16.39
ReVanced Patches 4.14.1 
ReVanced Integrations 1.14.1

### Bug description

When "Accessibility Control" is turned on, the bottom half of the seek bar is cut off, and in this state, the "tap on seek bar" feature cannot be used when opening the description or comments section.

reproduction
1. Turn on "Accessibility Control."
2. Restart the app.
3. Play any video.

### Device Environment

Android 14 One UI 6.1 Galaxy s23

### Acknowledgements

- [ ] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I didn't use any settings marked as `Experimental Flags`.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
It is also reproducible on unpatched YouTube 19.16.39

So this is an issue that Google needs to fix
